tests/http/test_client.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import respx
+from httpx import ConnectTimeout, Response, codes
+
+from mpt_api_client.http.client import MPTClient
+
+API_TOKEN = "test-token"
+API_URL = "https://api.example.com"
+
+
+@pytest.fixture
+def mpt_client():
+    return MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+
+def test_mpt_client_initialization():
+    client = MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+    assert client.api_token == API_TOKEN
+    assert client.base_url == API_URL
+
+
+def test_mpt_client_headers():
+    client = MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+    assert client.headers["Authorization"] == "Bearer test-token"
+    assert client.headers["User-Agent"] == "swo-marketplace-client/1.0"
```

Comment:
Umm...I believe we can join both tests, since headers are created as a part of the initialization. 

```suggestion
def test_mpt_client_initialization():
    client = MPTClient(base_url=API_URL, api_token=API_TOKEN)

    assert client.api_token == API_TOKEN
    assert client.base_url == API_URL
    assert client.headers["Authorization"] == "Bearer test-token"
    assert client.headers["User-Agent"] == "swo-marketplace-client/1.0"
    
```